src/main/groovy/io/seqera/wave/service/packages/PackagesCronJob.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Wave, containers provisioning service
+ *  Copyright (c) 2023-2024, Seqera Labs
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package io.seqera.wave.service.packages
+
+
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import io.micronaut.scheduling.TaskExecutors
+import io.micronaut.scheduling.annotation.ExecuteOn
+import io.micronaut.scheduling.annotation.Scheduled
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+
+/**
+ * Cron job to fetch packages
+ *
+ * @author Munish Chouhan <munish.chouhan@seqera.io>
+ */
+
+@CompileStatic
+@Singleton
+@Slf4j
+class PackagesCronJob {
```

Comment:
Let's make both the cron and controller optional and enabled only if the setting `wave.packages.enabled: true` is set

Reply:
done

Reply:
Make sure to use `wave.packages.` consistently. Still see some `wave.package.` around

Reply:
yes, its corrected